omnibus/cookbooks/omnibus-supermarket/files/default/ctl-commands/upgrade.rb

```diff
@@ -0,0 +1,18 @@
+# due to how things are being exec'ed, the CWD will be all wrong,
```

Comment:
Not relevant for this PR. But will be needed later when we want to upgrade supermarket.